cmd/root.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright © 2014-2020 Nimble
```

Comment:
```suggestion
Copyright © 2014-2021 Nimble
```

Reply:
Fixed in 4b74b60b785dc6e389b0ee1f9ae891de84fcbad6

Reply:
For this, please follow the updated [Git template repository](https://github.com/nimblehq/git-template): https://github.com/nimblehq/git-template/blob/main/LICENSE#L1

Reply:
Updated in 22d2254ae3f8e73cbab74d5725e62ad7a2a407b0